### DISCUSSION TITLE
[Suggestion] Restrict issues or at least issue write-access to project members

### DISCUSSION THREAD

**Discussion by rfc2822** [community/community#7421] [Projects and Issues]
(Similar to https://github.com/github/feedback/discussions/3679, but not necessarily connected to Discussions)

We have some public repos with project members who work on these projects and would like to use Issues for qualified (internal) bug reports, new feature tracking and so on.

However, when everyone can create issues, this causes a lot of mostly irrelevant issues (support requests although there are other support paths, unqualified bug reports etc.). So my suggestion: Please allow

* Issues to be either completely restricted to project members, or at least
* Issues to be read-only for everyone but project members (who have automatically write access); like the "Restrict editing to users in teams with push access only" option

**Comment by ernstki** [comment]
Oh my, please, yes.

I didn't think about it much until I read about [Daniel Stenberg's experiences](https://www.theregister.com/2025/05/07/curl_ai_bug_reports/) (he of [curl](https://github.com/curl/curl) fame), but it seems odd that we never had this ability before.

Especially now that discussions exist, which are good for accommodating "I can't get this to compile on my system" kinds of issues, and for a sufficiently-active community, can be used to [vet user-reported problems](https://hypr.land/news/issueChanges/) to determine if they merit developer time.
